tests/test_rotary_qkv_transform.py

```diff
@@ -0,0 +1,23 @@
+from modalities.models.gpt2.gpt2_model import RotaryTransform
+import torch 
+
+def test_rotary_transform():
+    bs = 10
+    n_heads = 4
+    embedding_dim = 128
+    seq_lenght = 12
+
+    q = torch.ones(bs,n_heads,seq_lenght, embedding_dim//n_heads) + 1
+    k = torch.ones(bs,n_heads,seq_lenght, embedding_dim//n_heads) + 2 
+    v = torch.ones(bs,n_heads,seq_lenght, embedding_dim//n_heads)
+
+    rotary_transform = RotaryTransform(n_embd=embedding_dim, n_head=n_heads)
+
+    q_rot, k_rot, v_rot = rotary_transform(q=q, k=k,v=v)
+
+    assert not torch.equal(q, q_rot)
+    assert q.shape == q_rot.shape
+    assert not torch.equal(k, k_rot)
+    assert k.shape == k_rot.shape
+    assert torch.equal(v, v_rot)
+    assert v.shape == v_rot.shape
```

Comment:
Ideally, create individual test for different asserts with descriptive test names.

Reply:
We refactored the test in 9b474f3c0b08ee704bd74121a993d00b8f79b24c.